src/mockServiceWorker.js

```diff
@@ -239,3 +253,11 @@ function ensureKeys(keys, obj) {
     return acc
   }, {})
 }
+
+function uuidv4() {
```

Comment:
I hope you don't mind that I'm commenting on a draft, but I think this can be removed and replaced with `utils/uuidv4`?.

Reply:
No, of course, I don't mind. 
The worker script is not transpiled, only copied as-is into the build directory. I suppose you suggest to reuse the same utility we have for internal usage, but that'd mean having to transpile the worker. I'm not strictly against that, but 1-1 representation of the script that will end up on developers' machines, as well as a guarantee of its integrity, is a reasonable advantage to consider against an extra transformation step.

Reply:
Thanks for the info, I wasn't aware of this.

Reply:
No worries. I agree that it may introduce certain code duplication, yet I'm unconvinced that the file should be built in any way. Plus, code reuse would still be dangerous, as the worker executes in a limited context, potentially making certain code un-reusable (i.e. if a code relies on `window` or `document`).